google-sheets/.env

```diff
@@ -0,0 +1,13 @@
+# description: The document ID for your Google Sheets spreadsheet (from its URL: https://docs.google.com/spreadsheets/d/[id]/edit)
```

Comment:
We're probably missing
```
# description: The path to the webhook
# configurable: false
TWILIO_SMS_WEBHOOK_URL=/log-sms
```

as this is the trigger on the UI that a twilio phone number is needed to be configured with the deployed webhook